src/layouts.jl

```diff
@@ -407,10 +405,10 @@ function update_child_bboxes!(layout::GridLayout, minimum_perimeter = [0mm,0mm,0
         # this is the minimum perimeter as decided by this child's parent, so that
         # all children on this border have the same value
         min_child_perimeter = [
-            c == 1  ? layout.minpad[1] : 0mm,
-            r == 1  ? layout.minpad[2] : 0mm,
-            c == nc ? layout.minpad[3] : 0mm,
-            r == nr ? layout.minpad[4] : 0mm
+            c == 1  ? layout.minpad[1] : pad_left[c],
```

Comment:
My best short explanation:
minimum padding should be propagated to all the children in the grid.

Additionally, perhaps I should add a clarifying comment that `minimum_perimeter` actually means `minimum_padding`. It took me a while to understand this. Should I?